src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -263,19 +263,20 @@ struct kds_cu_info
 
     result_type cuStats;
     // stats e.g.
-    // 0,vadd:vadd_1,0x1400000,0x4,0
-    // 1,vadd:vadd_2,0x1500000,0x4,0
-    // 2,mult:mult_1,0x1800000,0x4,0
+    // 0,0,vadd:vadd_1,0x1400000,0x4,0
+    // 0,1,vadd:vadd_2,0x1500000,0x4,0
+    // 0.2,mult:mult_1,0x1800000,0x4,0
```

Comment:
What are the trailing (0x4,0) for I see query::kds_cu_info having 4 entries, not 6?

Reply:
Got it, status and usage